bin/run-integration-tests.sh

```diff
@@ -45,18 +45,6 @@ if [ "${DRIVER}" = "SauceLabs" ]; then
   if [ -n "${PRIVACY}" ]; then CMD="${CMD} --capability public \"${PRIVACY}\""; fi
 fi
 
-# Browser Stack arguments
```

Comment:
Removing this as it was used for an evaluation that we didn't move forward with.